scripts/config.pl

```diff
@@ -131,12 +173,19 @@ sub disjunction_re {
     return '^(?:' . join('|', @_) . ')$';
 }
 my $exclude_re = disjunction_re(@excluded);
+my $exclude_non_full_re = disjunction_re(@excluded, @excluded_non_full);
 my $no_exclude_re = disjunction_re(@non_excluded);
 my $exclude_baremetal_re = disjunction_re(@excluded_baremetal);
 my $no_exclude_baremetal_re = disjunction_re(@non_excluded_baremetal);
+my $debug_features_re = disjunction_re(@debug_features);
+my $x509_features_re = disjunction_re(@x509_features);
+my $tls_features_re = disjunction_re(@tls_features);
+
 my %presets = (
                baremetal => {exclude_re => qr/($exclude_re|$exclude_baremetal_re)/,
                              no_exclude_re => qr/(?!$exclude_baremetal_re)$no_exclude_re|$no_exclude_baremetal_re/},
+               crypto_all => {exclude_re => qr/$exclude_non_full_re|$x509_features_re|$tls_features_re|$debug_features_re/,
+                              no_exclude_re => qr/^$/},
```

Comment:
1. Are you happy with the resulting config.h? I see we still have `MBEDTLS_ASN1_PARSE_C`, `MBEDTLS_ASN1_WRITE_C`, `MBEDTLS_PEM_PARSE_C`, `MBEDTLS_PK_PARSE_C`, `MBEDTLS_PK_C`, and `MBEDTLS_BASE64_C` enabled, for instance. Are those all really wanted?

2. Should we include the resulting config.h in our set of pre-made example config files (in `configs`)?